profile/profile.c

```diff
@@ -57,6 +57,7 @@ static void init(int frame_w, int frame_h)
         exit(1);
     }
 
+    ass_set_threads(ass_renderer, 0);
```

Comment:
At least `fuzz`, possibly all other utilities too, should opt into thread-safe logging

Reply:
Done.